.github/workflows/update_grammar.yml

```
@@ -26,8 +26,17 @@ jobs:
             - name: Download ANTLR
               working-directory: src/pytsql/grammar
               run: wget https://repo1.maven.org/maven2/org/antlr/antlr4/4.11.1/antlr4-4.11.1-complete.jar
-            - name: Download and re-generate
+            - name: Download latest grammar definition and re-generate grammar targets
               working-directory: src/pytsql/grammar
-              run: bash helper_generate_parsers.sh antlr4-4.11.1-complete.jar
+              run: bash helper_generate_grammar_targets.sh antlr4-4.11.1-complete.jar -d
             - name: Install package
               run: pip install -e .
```

Don't think you need the `-e` in here
```suggestion
              run: pip install .
```